NDTensors/src/Unwrap/src/functions/mul.jl

```diff
@@ -0,0 +1,4 @@
+function LinearAlgebra.mul!(CM::Exposed, AM::Exposed, BM::Exposed, α, β)
```

Comment:
```suggestion
function mul!(CM::Exposed, AM::Exposed, BM::Exposed, α, β)
```
since it looks like `mul!` is already imported.